buildpipeline/DotNet-CoreFx-Trusted-Windows.json

```diff
@@ -105,7 +105,7 @@
       },
       "inputs": {
         "filename": "$(Build.SourcesDirectory)\\corefx\\sync.cmd",
-        "arguments": "$(PB_SyncArguments) $(PB_OptionalToolingSyncArguments)",
+        "arguments": "$(PB_SyncArguments) $(PB_OptionalToolingSyncArguments) $(PB_PipelineBuildMSBuildArguments)",
```

Comment:
That happens when `variable` is undefined (which is different from empty string in VSTS). Setting it to empty string in the `variables` section is what lets it default to empty. In this build I have the empty string set:

    "PB_PipelineBuildMSBuildArguments": {
      "value": "",
      "allowOverride": true
    }

(I was already using this variable in the build before this PR, so that isn't showing up in the diff.)